code/modules/power/supermatter/supermatter_hypermatter_state.dm

```diff
@@ -0,0 +1,195 @@
+///Handles the main process of the hypermatter
+/obj/machinery/power/supermatter_crystal/proc/handle_hypermatter_state()
+
+	if(!anchored) //Don't unanchor the shards
+		damage += 150
+		hypermatter_state = FALSE
+		update_appearance()
+		return
+
+
+	var/static/list/angles = list(0, 45, 90, 135, 180, 225, 270, 315, 360)
+
```

Comment:
applies here too, also what do these angles represent exactly? name does nothing to explain

Reply:
the angles are for the nuclear particles to exit at those specific ones. I'll change the name to better reflect that